physical/common.go

```diff
@@ -0,0 +1,10 @@
+package physical
+
+func appendIfMissing(slice []string, i string) []string {
```

Comment:
Done. I totally missed the helper/strutil package when I did this originally. This is much better.